frame/support/test/tests/benchmark_ui/dup_extrinsic_call.stderr

```diff
@@ -0,0 +1,5 @@
+error: Only one #[extrinsic_call] or #[block] attribute is allowed per benchmark.
+  --> tests/benchmark_ui/dup_extrinsic_call.rs:14:3
+   |
+14 |         #[extrinsic_call]
+   |         ^
```

Comment:
What would be really awesome as a nice-to-have would be to have it show where the first attribute is. This is currently only showing where the 2nd one is.

Reply:
I worked in the Rust compiler before, so it's possible that syn doesn't currently support adding multiple spans to an error, and if so, forget about my suggestion above.

Reply:
Yeah as far as I can tell you can only pass 1 span